### PR TITLE
develop/hardware: Update I2C example in the i2c-and-spi section

### PR DESCRIPTION
**Pull request by acostach**
The existing sample application is now old, let's use the balena-sense project as an example for I2C.

Change-type: patch

---

> *Please make sure to read the [CONTRIBUTING](https://github.com/balena-io/docs/blob/master/CONTRIBUTING.md) document before opening the PR for relevant information on contributing to the documentation. Thanks!*

### CODE REVIEW

File: pages/learn/develop/hardware/i2c-and-spi.md [hunk 1]
```diff
@@ -30,26 +30,28 @@ Many sensors and peripherals use either the [I²C (Inter-Integrated Circuit)][i2
...
+
+```
+services:
+  myservice:
+  ...
+    privileged:true
+```
 
-__Note:__ A few places will talk about adding the modules to the /etc/modules file so that they are there on boot. This will not work on the {{ $names.company.lower }} system because that file is not mapped to the host OS.
+Single-container fleets run in privileged modeby default. If security and sandboxing are required then a multicontainer release should be created, even if only a single service is required.
```
Comment (lurch) [inline pages/learn/develop/hardware/i2c-and-spi.md:52]: "modeby" -> "mode by" ?

Comment (acostach) [reply pages/learn/develop/hardware/i2c-and-spi.md]: Fixed, thanks for spotting that typo

**Review by lmbarros — CHANGES_REQUESTED** [review]
This looks great to me Alexandru! The CI checks are complaining that `myservice` is an unknown word :roll_eyes: . I added comments with suggestions that will hopefully make this check pass.

**Comment by lmbarros** [comment]
LGTM

**Pull request merged**